### DISCUSSION TITLE
2006/07/24/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-1-labview-%E6%98%AF%E8%87%AA%E5%8A%A8%E5%A4%9A%E7%BA%BF%E7%A8%8B%E8%AF%AD%E8%A8%80

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#98] [Announcements]
# 2006/07/24/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-1-labview-%E6%98%AF%E8%87%AA%E5%8A%A8%E5%A4%9A%E7%BA%BF%E7%A8%8B%E8%AF%AD%E8%A8%80

一. LabVIEW 是自动多线程语言

https://blog.qizhen.xyz/2006/07/24/labview-%E7%A8%8B%E5%BA%8F%E4%B8%AD%E7%9A%84%E7%BA%BF%E7%A8%8B-1-labview-%E6%98%AF%E8%87%AA%E5%8A%A8%E5%A4%9A%E7%BA%BF%E7%A8%8B%E8%AF%AD%E8%A8%80

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Unknown
2006年09月18日 15:29 编辑
这种多线程只是一种假象，其实用不用都无所谓，这只是LabVIEW让人看起来是这个样子，其实根本不是这个样子

回复
竹贤
2006年07月27日 18:37 编辑
请抓紧写啊。我等可以认真拜读。

回复
Qizhen
2006年07月27日 18:04 编辑
我觉得 8.0 比 7.1 最大的改进就是多了 Project, Library 等来帮助用户管理大型程序。这些新特性也的确极大的方便了 LabVIEW 开发大型程序。不过要一一细说，我也要花一段时间。等我想好了再写下来。

回复
竹贤
2006年07月27日 17:27 编辑
LabVIEW 8.0与7.0版本有较大的不同，现在也买不到8.0的书籍，请大虾介绍一下如何使用8.0编程（与以前不同之处）。

回复
Qizhen
2006年07月26日 18:44 编辑
我的工作保密 :)
 
但是想和我联系，直接在这里留言就可以了。

回复
润
2006年07月26日 15:04 编辑
你好,我是来自中国测控网www.ck365.cn“能认识下吗?:)

回复
清扬
2006年07月26日 12:17 编辑
不错,很有启发.
你是NI的工程师吗?
